src/writeData/components/WriteDataHelperBuckets.tsx

```diff
@@ -68,17 +70,29 @@ const WriteDataHelperBuckets: FC<Props> = ({
     body = (
       <List
         backgroundColor={InfluxColors.Grey5}
-        style={{height: '200px'}}
+        style={
+          disabled
+            ? {
+                pointerEvents: 'none',
+                height: '200px',
+                cursor: 'not-allowed',
+                borderColor: '#1a1a2a',
+                backgroundColor: '#1a1a2a',
+                opacity: '0.5',
+                color: '#828294',
+              }
+            : {height: '200px'}
+        }
```

Comment:
It looks like the [List](https://influxdata.github.io/clockface/?path=/story/components-list-family--list) component will handle this styling for you if you pass `disabled={disabled}` into it

Reply:
ah - thanks! I'll definitely use that I tried handing it into List and that wasn't a prop that exists but handing it into List.Item does the trick.